wolfssl/internal.h

```diff
@@ -4712,7 +4836,10 @@ enum ContentType {
     change_cipher_spec = 20,
     alert              = 21,
     handshake          = 22,
-    application_data   = 23
+    application_data   = 23,
+#ifdef WOLFSSL_DTLS13
+    ack                = 25,
```

Comment:
This should be 26. It is only refered to as 25 in
https://www.ietf.org/archive/id/draft-ietf-tls-dtls13-43.html#ack-msg
But in the following it is proposed as 26
https://www.ietf.org/archive/id/draft-ietf-tls-dtls13-43.html#name-demultiplexing-dtls-records
https://www.ietf.org/archive/id/draft-ietf-tls-dtls13-43.html#name-iana-considerations
This is an error in the draft rfc by the way. You should report it to them.

Reply:
Ok. So its settled that this will have value 26.